Realm/RLMRealm.h

```diff
@@ -551,6 +551,19 @@ typedef void (^RLMMigrationBlock)(RLMMigration *migration, NSUInteger oldSchemaV
 + (NSUInteger)schemaVersionAtPath:(NSString *)realmPath encryptionKey:(NSData *)key error:(NSError **)error;
 
 /**
+ Returns whether or not the realm at the given path can be accessed without performing a migration.
+
+ @param realmPath   Path to a Realm file.
+ @param key         64-byte encryption key.
```

Comment:
"64-byte encryption key or nil for unencrypted Realms."

It's not optional most of the places where there's a key parameter, so it needs to call out that this one is.